content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -145,7 +145,7 @@ voluntary evictions, not all causes of unavailability.
 {{< /note >}}
 
 A `maxUnavailable` of 0% (or 0) or a `minAvailable` of 100% (or equal to the
-number of replicas) may block node drains entirely. This is permitted as per the 
+number of replicas) will block node drains entirely. This is permitted as per the 
```

Comment:
With the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) in mind, how about:
```markdown
If you set `maxUnavailable` to 0% or 0, or you set `minAvailable` to 100% or the number of replicas,
you are requiring zero voluntary evictions. When you set zero voluntary evictions for a workload
object such as ReplicaSet, then you cannot successfully drain a Node running one of those Pods.
If you try to drain a Node where an unevictable Pod remains running, the drain never completes.
```

What do you think @manishbansal8843 ?

Reply:
hey, @manishbansal8843. would you update this thread? thanks.

Reply:
hi @sftim yes, we can write that way as well. Can you please point me to the section of style guide so that i can also keep the same in mind for future reference. I am really not sure how one way of writing is better than other. thanks.

Reply:
@manishbansal8843 take a look at https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices

Reply:
@manishbansal8843 take a look at https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices